output.log

```diff
@@ -0,0 +1,143 @@
+[INFO] Scanning for projects...
```

Comment:
Oops, I forgot to merge this before pushing new code... This push was just for the sake of saving the code while it works. Definitely not ready for the pull request, sorry for that. Do you know if there is anyway to merge the first code without the last commit?

Reply:
Honestly I would just google "github remove commit from pull request" since I'm not really a GitHub expert. This link looks pretty promising: https://stackoverflow.com/questions/36168839/how-to-remove-commits-from-pull-request

Reply:
I just finished this part of the code and committed again